webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/Components/WebhooksTable/Components/EnabledCell.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const EnabledCell = ({ condition }) =>
+  condition ? <span className="fa fa-check" /> : null;
```

Comment:
I think not showing an icon when it's not enabled might make it look like the icon didn't load. Instead of null can there be an X or something similar? Also this should be an `Icon` component from Patternfly